### PR TITLE
fix(tester.py): ignore cdc stream errors

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/z2WCnCHa/4156-fixtesterpy-ignore-cdc-stream-errors

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by roydahan** [comment]
This  is only a workaround for 2021.1 upgrade from 2020.1.
We shouldn't ignore this on master or the general case.
I had a question if we can do this only when the base version is 2020.1.

**Comment by dkropachev** [comment]
> This is only a workaround for 2021.1 upgrade from 2020.1. We shouldn't ignore this on master or the general case. I had a question if we can do this only when the base version is 2020.1.

fixed

File: upgrade_test.py [hunk 1]
```diff
@@ -111,6 +112,11 @@ class UpgradeTest(FillDatabaseData):
     insert_rows = None
     truncate_entries_flag = False
 
+    def __init__(self, *args):
+        super().__init__(*args)
+        if not self.params.get('scylla_version') == '2020.1':
```
Comment (roydahan) [inline upgrade_test.py:117]: If not?

Comment (dkropachev) [reply upgrade_test.py]: removed

**Review by bentsi — APPROVED** [review]

**Pull request merged**